omnibus/resources/fips/openssl.cnf

```diff
@@ -0,0 +1,15 @@
+config_diagnostics = 1
+openssl_conf = openssl_init
+
+# TODO: template install path
+.include /opt/datadog-agent/embedded/ssl/fipsmodule.cnf
+
+[openssl_init]
+providers = provider_sect
+
+[provider_sect]
+fips = fips_sect
+base = base_sect
+
+[base_sect]
+activate = 1
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Can you add a new line?